DSharpPlus.Commands/Processors/BaseCommandProcessor/BaseCommandProcessor.cs

```diff
@@ -223,6 +223,30 @@ await this.extension.commandErrored.InvokeAsync(converterContext.Extension, new
                 parsedArguments.Add(converterContext.Parameter, optional.RawValue);
             }
 
+            if (parsedArguments.Count == 0)
+            {
+                foreach (CommandParameter parameter in converterContext.Command.Parameters)
+                {
+                    if (!parameter.DefaultValue.HasValue)
+                    {
+                        await this.extension.commandErrored.InvokeAsync(converterContext.Extension, new CommandErroredEventArgs()
+                        {
+                            Context = CreateCommandContext(converterContext, eventArgs, parsedArguments),
+                            Exception = new ArgumentParseException(converterContext.Parameter, null, $"Argument Converter for type {converterContext.Parameter.Type.FullName} was unable to parse the argument."),

```

Comment:
> $"Argument Converter for type {converterContext.Parameter.Type.FullName} was unable to parse the argument."

Try to give the most amount of information possible to the dev. The dev is already receiving a `ArgumentParseException`, it's pretty clear that the argument parser was unable to parse the argument. What about:

> $"Not enough argument data to parse {converterContext.Parameter.Name}"